src/views/Events/index.js

```diff
@@ -254,68 +288,63 @@ export default class Events extends Component {
       content = <EventList events={this.state.filteredEvents} />;
     }
 
+    let eventsAtGordon = [];
+
+    const handleChange = (event) => {
```

Comment:
One reason you might be having issues is because I don't think you bound this function anywhere. Because of some complicated stuff that gets into the worst parts of Javascript, methods in a React class component have to be bound to the class component. See lines 42-44 above for examples.

Reply:
https://github.com/gordon-cs/gordon-360-ui/pull/1015/commits/0990cff23f671494e1aa8835dfacbca3d2b2b1d6